mlflow/ml-package-versions.yml

```diff
@@ -749,6 +749,10 @@ langchain:
           "langchain-huggingface",
           # Install langchain-databricks from the main branch
           "git+https://github.com/langchain-ai/langchain-databricks.git@main#egg=langchain-databricks&subdirectory=libs/databricks",
+          # Required for testing uc tools
+          "git+https://github.com/unitycatalog/unitycatalog.git#subdirectory=ai/core",
```

Comment:
When do we release this?

Reply:
Maybe this week if Ben gets approval from Xiangrui :)

Reply:
Can you add a TODO comment?